### PR TITLE
feat: amplitude analytics in explore, and make entire token row clickable

### PR DESCRIPTION
**Pull request by kayleegeorge**
Relevant Jiras: https://uniswaplabs.atlassian.net/browse/WEB-685 , https://uniswaplabs.atlassian.net/browse/WEB-205?atlOrigin=eyJpIjoiMWNjZjZiYTExYWRiNDE5YmI2MjliZmMxNzYzOTA5YzkiLCJwIjoiaiJ9

Screen Shot 2022-08-02 at 3 20 01 PM
Screen Shot 2022-08-02 at 3 20 21 PM

### CODE REVIEW

**Comment by lynnshaoyu** [comment]
looks like there is a deploy / build error on vercel? @kayleegeorge

File: src/pages/Explore/index.tsx [hunk 1]
```diff
@@ -53,23 +55,25 @@ const FiltersWrapper = styled.div`
...
-        <FiltersContainer>
-          <NetworkFilter />
-          <FavoriteButton />
-          <TimeSelector />
-        </FiltersContainer>
-        <SearchContainer>
-          <SearchBar />
-        </SearchContainer>
-      </FiltersWrapper>
+    <Trace page={PageName.POOL_PAGE} shouldLogImpression>
```
Comment (lynnshaoyu) [inline src/pages/Explore/index.tsx:58]: should be Explore page (not Pool page)

File: src/pages/App.tsx [hunk 2]
```diff
@@ -4,7 +4,8 @@ import { Trace } from 'components/AmplitudeAnalytics/Trace'
 import Loader from 'components/Loader'
 import TopLevelModals from 'components/TopLevelModals'
 import ApeModeQueryParamReader from 'hooks/useApeModeQueryParamReader'
-import { lazy, Suspense, useEffect } from 'react'
+import { lazy, Suspense } from 'react'
```
Comment (lynnshaoyu) [inline src/pages/App.tsx:7]: we can keep useEffect in the same import line (7) as it was before?

File: src/pages/Explore/index.tsx (hunk 1 above)
Comment (kayleegeorge) [reply src/pages/Explore/index.tsx:58]: OH RIP, must have changed when I merged -- thanks good catch!

**Comment by lynnshaoyu** [comment]
also, could you try to add some screenshots in the PR description of the amplitude events being logged to console on your local build? similar to screenshots in here: https://github.com/Uniswap/interface/pull/4151

just to check they're being logged on the correct page / user triggered event. thank you!!

**Review by JFrankfurt — APPROVED** [review]

**Pull request merged**